src/geometry.cpp

```diff
@@ -79,4 +81,25 @@ namespace geometry {
 	  return v;
 
 	}
+
+	std::vector<Cell> raytraceLine_conversion(const Cell& start, const Cell& end) {
+
+	  // Get the interpolated pixels bresenham
+	  const auto pixelsAsInts = bresenham_conversion(start.x, start.y, end.x, end.y);
+
+	  // Convert std::vector<std::vector<int>> to std::vector<Cell>
+	  std::vector<Cell> pixelsAsCells;
+
+	  for (auto const &pixel:pixelsAsInts) {
+	    Cell temp;
+	    temp.x = pixel[0];
+	    temp.y = pixel[1];
+	    pixelsAsCells.push_back(temp);
```

Comment:
Since this is C++17, you can do
```
pixelsAsCells.emplace_back({pixel[0], pixel[1]});
```
though, can you see if 
```
pixelsAsCells.emplace_back({pixel});
```
or
```
pixelsAsCells.emplace_back(pixel);
```
works?

Reply:
Surprisingly, it likes none of them? All of them throw an error.

Reply:
Huh, I tried it myself and also got compile errors (I assume you didn't get runtime errors).
Looks like there's an ambiguity in casting with the above which leads to the need to cast it to a `std::initializer_list<int>` or something similar, which kind of defeats the purpose in my mind.
I got it to work in the conventional way

https://onlinegdb.com/SJmVj_N-P

Reply:
Done.